src/main/fc/tasks.c

```diff
@@ -409,6 +410,10 @@ task_attribute_t task_attributes[TASK_COUNT] = {
 #ifdef USE_CRSF_V3
     [TASK_SPEED_NEGOTIATION] = DEFINE_TASK("SPEED_NEGOTIATION", NULL, NULL, speedNegotiationProcess, TASK_PERIOD_HZ(100), TASK_PRIORITY_LOW),
 #endif
+
+#ifdef USE_TELEMETRY_SBUS2
+    [TASK_TELEMETRY_SBUS2] = DEFINE_TASK("SBUS2_TELEMETRY", NULL, NULL, taskSendSbus2Telemetry, TASK_PERIOD_US(125), TASK_PRIORITY_LOW),
```

Comment:
8kHz task speed is very fast. Is it necessary to run this fast? It might be a problem for F4, maybe even for F7.

Reply:
Priority is low. The sbus2 telemetry has somewhat tight timing requirements.

2ms dead time, follow by 8 telemetry slots of ~650us each, but a bit of dead time is needed between frames.

If we lower too much, we may be too late to send telemetry data, but we risk breaking the timing requirements.

It is probably a good idea to limit it to f7 and h7 anyway, since sbus is an inverted protocol and would need extra hardware on f4.

Reply:
I experimented again with this. 8k is required for all sensors to work.

If I lower it to 4k, the last telemetry slotsbecome unreliable and often don't update.

I did lower the priority to LOWEST and limited it to F7, H7 and G4 mcus since the F4 needs extra circuitry to use a half-duplex inverted uart.